client/src/app/dashboard/dashboard-overview/dashboard-overview.component.html

```diff
@@ -19,18 +19,175 @@
 <div class="container-fluid">
   <div class="row">
     <div class="col-md-3 col-sm-4">
-    <app-dashboard-card route="crawler" componentType={{crawler}} title="Crawler" img="../assets/images/crawler.png"></app-dashboard-card>
+      <app-dashboard-card route="crawler" componentType={{crawler}} title="Crawler" img="../assets/images/crawler.png" collapse_target="#demo1" numberOfNotification="3"></app-dashboard-card>
     </div>
     <div class="col-md-3 col-sm-4">
-    <app-dashboard-card route="webApi" componentType={{webApi}} title="Web API" img="../assets/images/webapi.png"></app-dashboard-card>
+      <app-dashboard-card route="webApi" componentType={{webApi}} title="Web API" img="../assets/images/webapi.png" collapse_target="#demo2" numberOfNotification="1"></app-dashboard-card>
     </div>
     <div class="col-md-3 col-sm-4">
-    <app-dashboard-card route="webapp" componentType={{webApp}} title="Web APP" img="../assets/images/webapp.png"></app-dashboard-card>
+      <app-dashboard-card route="webapp" componentType={{webApp}} title="Web APP" img="../assets/images/webapp.png" collapse_target="#demo3" numberOfNotification="2"></app-dashboard-card>
     </div>
     <div class="col-md-1 col-sm-4">
     </div>
     <div class="col-md-2 col-sm-4">
-    <app-statuscard></app-statuscard>
+      <app-statuscard></app-statuscard>
+    </div>
   </div>
+
+  <!--Begin notification pannels-->
+  <div id="demo1" class="collapse">
```

Comment:
please use a more angular like way of doing this. best would most likely be to use a ng-if (https://angular.io/api/common/NgIf)